src/main/components/form/validator.ts

```diff
@@ -300,3 +300,17 @@ export const isValidCurrency: Validator = value => {
     return 'minLengthRequired';
   }
 };
+
+let otherGenderTitle: string | string[];
+
+export const validateGenderTitle: Validator = value => {
+  otherGenderTitle = value;
+};
+
+export const validatePreferredOther: Validator = value => {
+  if (otherGenderTitle === 'Other') {
+    if (/^\d+$/.test(value as string) || /[0-9]/.test(value as string) || (value as string).trim().length === 0) {
```

Comment:
I think you need to split out the number check from the required validation as it needs to display a different error message